lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -14,6 +14,8 @@
 from freezegun import freeze_time
 
 from django.conf import settings
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import UsageKey
```

Comment:
@asadazam93 , please follow the correct way of ordering imports see: https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

1. standard library imports
2. related third party imports
3. local application/library specific imports

You should put a blank line between each group of imports.